Cargo.toml

```
@@ -196,161 +196,161 @@ rust-version = "1.83.0"
+proc-macro2 = "1"
+prometheus = "0.13"
+proptest = { version = "1", default-features = false }
 prost = { version = "0.12", default-features = false }
 prost-build = "0.12"
-protobuf = "2.27.1"
-protoc-grpcio = "3.0.0"
-quote = "1.0.2"
-r2d2 = "0.8.10"
-rand = { version = "0.8.5", default-features = false }
+protobuf = "2"
+protoc-grpcio = "3"
+quote = "1"
+r2d2 = "0.8"
+rand = { version = "0.8", default-features = false }
 rand_chacha = "0.3"
-rand_core = { version = "0.6.4", default-features = false }
+rand_core = { version = "0.6", default-features = false }
 rand_hc = { version = "0.3", default-features = false }
-rayon = "1.9"
+rayon = "1"
 regex = "1"
 reqwest = { version = "0.11", default-features = false }
-retry = "2.0"
-rjson = "0.3.1"
-rocket = "0.5.1"
-rusoto_core = { version = "0.48.0", default-features = false }
-rusoto_s3 = { version = "0.48.0", default-features = false }
+retry = "2"
+rjson = "0.3"
+rocket = "0.5"
+rusoto_core = { version = "0.48", default-features = false }
+rusoto_s3 = { version = "0.48", default-features = false }
 schnorrkel-og = { version = "0.11.0-pre.0", default-features = false }
-scoped_threadpool = "0.1.*"
+scoped_threadpool = "0.1"
 secrecy = "0.8"
-semver = "1.0"
+semver = "1"
 sentry = { version = "0.32", default-features = false }
-serde = { version = "1.0.197", default-features = false }
-serde_cbor = { version = "0.11.1", default-features = false }
-serde_derive = "1.0"
-serde_json = { version = "1.0.114", default-features = false }
-serde_with = { version = "3.6", default-features = false }
-serial_test = "3.0"
-sha2 = { version = "0.10.8", default-features = false }
+serde = { version = "1", default-features = false }
+serde_cbor = { version = "0.11", default-features = false }
+serde_derive = "1"
+serde_json = { version = "1", default-features = false }
+serde_with = { version = "3", default-features = false }
+serial_test = "3"
+sha2 = { version = "0.10", default-features = false }
 sha3 = { version = "0.10", default-features = false }
 signal-hook = "0.3"
-signature = { version = "2.1.0", default-features = false }
-siphasher = "1.0"
+signature = { version = "2", default-features = false }
+siphasher = "1"
 slip10_ed25519 = "0.1"
-slog = { version = "2.7", default-features = false }
-slog-async = "2.8"
-slog-atomic = "3.1"
-slog-envlogger = "2.2"
-slog-json = "2.6"
-slog-scope = "4.4.0"
-slog-stdlog = "4.1.1"
-slog-term = "2.9"
-static_assertions = "1.1.0"
+slog = { version = "2", default-features = false }
+slog-async = "2"
+slog-atomic = "3"
+slog-envlogger = "2"
+slog-json = "2"
+slog-scope = "4"
+slog-stdlog = "4"
+slog-term = "2"
+static_assertions = "1"
 stdext = "0.3"
-subtle = { version = "2.4.1", default-features = false }
-syn = "2.0"
-tempfile = "3.10.1"
+subtle = { version = "2", default-features = false }
+syn = "2"
+tempfile = "3"
 textwrap = "0.16"
-tiny-bip39 = "1.0"
-tokio = "1.36"
+tiny-bip39 = "1"
+tokio = "1"
 toml = "0.8"
-url = "2.5.0"
+url = "2"
 walkdir = "2.4"
```

nor sure if this was left intentionally
```suggestion
walkdir = "2"
```